browser_use/browser/browser.py

```diff
@@ -302,34 +346,47 @@ async def _setup_browser(self, playwright: Playwright) -> PlaywrightBrowser:
 			raise
 
 	async def close(self):
-		"""Close the browser instance"""
-		if self.config.keep_alive:
-			return
-
+		"""Close the browser."""
 		try:
-			if self.playwright_browser:
-				await self.playwright_browser.close()
-				del self.playwright_browser
-			if self.playwright:
-				await self.playwright.stop()
-				del self.playwright
-			if chrome_proc := getattr(self, '_chrome_subprocess', None):
-				try:
-					# always kill all children processes, otherwise chrome leaves a bunch of zombie processes
-					for proc in chrome_proc.children(recursive=True):
-						proc.kill()
-					chrome_proc.kill()
-				except Exception as e:
-					logger.debug(f'Failed to terminate chrome subprocess: {e}')
+			if not self.config._force_keep_browser_alive:
```

Comment:
```suggestion
			if not self.config.keep_alive:
```
This config was renamed, make sure you are working off the latest `main` branch